.actions/assistant.py

```diff
@@ -60,6 +60,23 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     return py
 
 
+def _retrieve_files(directory: str, *ext: str):
+    subfolders, files = [directory], []
+
+    ext = tuple(e.lower() for e in ext)
```

Comment:
Do we have capital extensions anywhere?

Reply:
because globbing with a wildcard only is error prone and slower.

Reply:
I do not think we are here playing with ms :)
We can change it later on...